It seems that tests are failing even in master. 

```
=== RUN   TestBadValues
--- FAIL: TestBadValues (0.00s)
panic: assignment to entry in nil map [recovered]
        panic: assignment to entry in nil map

--- a/pkg/lint/lint_test.go
+++ b/pkg/lint/lint_test.go
@@ -39,7 +39,7 @@ func TestBadChart(t *testing.T) {
 	var w, e, e2, e3 bool
 	for _, msg := range m {
 		if msg.Severity == support.WarningSev {
-			if strings.Contains(msg.Text, "Templates directory not found") {
+			if strings.Contains(msg.Text, "Directory 'templates/' not found") {
 				w = true
 			}
 		}

--- a/pkg/lint/rules/chartfile.go
+++ b/pkg/lint/rules/chartfile.go
@@ -59,7 +59,7 @@ func Chartfile(linter *support.Linter) {
 func validateChartYamlFileExistence(chartPath string) (lintError support.LintError) {
 	_, err := os.Stat(chartPath)
 	if err != nil {
-		lintError = fmt.Errorf("Chart.yaml file does not exists")
+		lintError = fmt.Errorf("Chart.yaml file does not exist")
 	}
 	return
 }
@@ -137,7 +137,7 @@ func validateChartEngine(cf *chart.Metadata) (lintError support.LintError) {
 		keys = append(keys, str)
 	}
 
-	lintError = fmt.Errorf("Chart.yaml: 'engine %v not valid. Valid options are %v", cf.Engine, keys)
+	lintError = fmt.Errorf("Chart.yaml: engine '%v' not valid. Valid options are %v", cf.Engine, keys)
 	return
 }
 

--- a/pkg/lint/rules/template.go
+++ b/pkg/lint/rules/template.go
@@ -110,7 +110,7 @@ func Templates(linter *support.Linter) {
 // Validation functions
 func validateTemplatesDir(templatesPath string) (lintError support.LintError) {
 	if fi, err := os.Stat(templatesPath); err != nil {
-		lintError = fmt.Errorf("Templates directory not found")
+		lintError = fmt.Errorf("Directory 'templates/' not found")
 	} else if err == nil && !fi.IsDir() {
 		lintError = fmt.Errorf("'templates' is not a directory")
 	}

